components/questionaire.js

```diff
@@ -38,6 +40,14 @@ export class Questionaire extends Component {
       jsx_array.push(<h3 key={k1 + "-h3"}>{section_name}</h3>);
       questions
         .filter(q => q.section_name === section_name)
+        .filter(g =>
+          evaluateRowConditions(
+            g,
+            reduxState.questions,
+            reduxState.multiple_choice_options,
```

Comment:
you could just pass in redux state

Reply:
oh maybe that would make it harder to use the fixtures in testing

Reply:
yeah, that was my thinking, essentially